pkg/api/types.go

```diff
@@ -686,6 +687,12 @@ const (
 	StorageMediumHugePages StorageMedium = "HugePages" // use hugepages
 )
 
+var ValidStorageMedia = sets.NewString(
+	string(StorageMediumDefault),
+	string(StorageMediumMemory),
+	string(StorageMediumHugePages),
+)
+
```

Comment:
this doesnt look like it should go in types.go

Reply:
Good point! Addressed. PTAL. Thanks!